packages/edit-post/src/components/text-editor/style.scss

```diff
@@ -1,6 +1,8 @@
 .edit-post-text-editor {
 	position: relative;
 	width: 100%;
+	background-color: #fff;
```

Comment:
Should this and another `#fff` in this PR use the `$white` variable? 

https://github.com/WordPress/gutenberg/blob/b3f900c34d41c282db5a6d5a43f8eaf5f091f3d6/packages/base-styles/_colors.scss#L26